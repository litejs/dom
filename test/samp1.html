<!DOCTYPE html>
<html lang=en data-a="http://www.w3.org/1999/xhtml" data-b="http://opengraphprotocol.org/schema/"><head>
<title>1 + 2 = 3</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<style type="text/css">
/*<![CDATA[*/
body { background-image: url("marble.png?width=300&height=300") }
/*]]>*/
</style>
<link rel="stylesheet" type="text/css" href="styles/a.css?123">
<!--[if lt IE 8]><link rel="stylesheet" type="text/css" href="styles/ie.css?123"><![endif]-->
<script type="text/javascript" async="" src="http://www.google-analytics.com/ga.js"></script>
<script type="text/javascript" >
//<![CDATA[
document.write("<");
//]]>
</script>
<script type="module" src="main.js"></script> <script nomodule src="fallback.js"></script>
<style type="text/css">.class{position:absolute;}</style></head>
<body onload="
	alert('&quot;code&quot;');
" class="reset page"><!-- table --!>
    <ul class="gr">
            <li class="home"><span title="&amp;&lt;&gt;&apos;&quot;">&amp;&lt;&gt;&apos;&quot;</span ><img src="home.gif" alt="Home"></li>
    </ul>
        <div id="header" class="tpl-header"><!--My favorite operators are > and <!-->
	<iframe style="display:none" tabindex="-1"></iframe>
	</div>
</body>
</html>
</invalid>
