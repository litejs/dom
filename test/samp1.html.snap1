<!DOCTYPE html><html lang=en data-a=http://www.w3.org/1999/xhtml data-b=http://opengraphprotocol.org/schema/ ><head><title>1 + 2 = 3</title><meta http-equiv=Content-Type content="text/html; charset=utf-8"><style>
/**/
body { background-image: url("CDATA-in-block-comment.png?width=300&height=300") }
/**/
</style><link rel=stylesheet type=text/css href=styles/a.css?123><script>
document.write('<p>ETAGO delimiter</p><script>console.log("A")<\/script><script>console.log("B")<\/script>')
</script><script async src=http://www.google-analytics.com/ga.js></script><script>
//
document.write("< CDATA with line comment");
if (a > 12) throw "a <\!---->not <\!-- --> 12"
//
</script><script type=module src=main.js></script><script nomodule src=fallback.js></script><style>.class{position:absolute;}</style></head><body onload="console.log('&quot;code&quot;')" class="reset page"><ul class=gr style="background-image:url('https://1.1.1.1:80/i.png');height:365px"><li class="a clc b">Some<b>bold</b>text  here</li><li class=home><span title="&&lt;>'&quot;">&&lt;>'"</span><img src=home.gif alt=Home></li></ul><div id=header class=tpl-header><iframe style=display:none tabindex=-1></iframe></div></body></html>