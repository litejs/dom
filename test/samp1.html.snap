<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xmlns:og="http://opengraphprotocol.org/schema/"><head>
<title>1 + 2 = 3</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<style type="text/css">
/*<![CDATA[*/
body { background-image: url("marble.png?width=300&height=300") }
/*]]>*/
</style>
<link rel="stylesheet" type="text/css" href="styles/a.css?123">
<!--[if lt IE 8]><link rel="stylesheet" type="text/css" href="styles/ie.css?123"><![endif]-->
<script type="text/javascript" async src="http://www.google-analytics.com/ga.js"></script>
<script type="text/javascript">
//<![CDATA[
document.write("<");
//]]>
</script>
<style type="text/css">.class{position:absolute;}</style></head>
<body onload="alert('&quot;code&quot;');" class="reset page">
    <ul class="gr">
            <li class="home"><img src="home.gif" alt="Home"></li>
    </ul>
        <div id="header" class="tpl-header"><!--My favorite operators are > and <!-->
	<table cellpadding="0" cellspacing="0" border="0"><!-- table -->
		<tbody><tr>
				<td>A</td>
				<td>B</td>
		</tr></tbody>
	</table>
	</div>
</body>
</html>

